test/expected.txt

```diff
@@ -19,6 +19,8 @@
 || Example: find alice bob charlie
 || list: Displays all persons in the address book as a list with index numbers.
 || Example: list
+|| sort: Sort all persons in the address book in alphabetical order.
+|| Example: sort
```

Comment:
Where have you added the tests for sort i.e. the expected outputs?

Reply:
Added the test cases. Travis Build failed the test cases but program works fine in IDE...

Reply:
I have reverted back to the original test cases but travis build still fails... is there something wrong with my formatting?